Dockerfile.ceph

```diff
@@ -88,6 +102,12 @@ USER ceph
 WORKDIR /ceph
 ADD --chown=ceph:ceph --chmod=755 https://raw.githubusercontent.com/ceph/ceph/v$CEPH_CLUSTER_VERSION/src/vstart.sh .
 
+COPY <<EOF ./CMakeCache.txt
+ceph_SOURCE_DIR:STATIC=/ceph
+WITH_MGR_DASHBOARD_FRONTEND:BOOL=ON
+WITH_RBD:BOOL=ON
+EOF
```

Comment:
vstart assumes that Ceph is always run from sources, and hence the CMake artifacts are over there. We should probably override that if some `WITH_MGR_DASHBOARD_FRONTEND` env var is defined.